Storage/Rust/RustLogins.swift

```diff
@@ -501,8 +501,8 @@ public class RustLogins {
 
         queue.async {
             guard self.isOpen else {
-                let error = LoginsStoreError.MismatchedLock(message: "Database is closed")
-                deferred.fill(Maybe(failure: error as MaybeErrorType))
+                //let error = LoginsStoreError.MismatchedLock(message: "Database is closed")
```

Comment:
@tarikeshaq the changelog says:

> Note that some errors were removed, which is technically a breaking change, but none of our consumers use those errors so it's not a breaking change in practice.

Guess that wasn't really correct. It's in use here and fails the build now. I have no idea what to do, can you provide the changes needed here?